src/main/resources/assets/immersiveengineering/manual/autoload.json

```diff
@@ -0,0 +1,99 @@
+{
+  "general": {
+    "category_weight": 0,
+    "entry_list": [
+      {
+        "source": "introduction",
+        "weight": -1
+      },
+      "hemp",
+      "ores",
+      "alloys",
+      "components",
+      "plates",
+      "craftingtable",
+      "workbench",
+      "alloykiln",
+      "cokeoven",
+      "crude_blast_furnace",
+      "improved_blast_furnace",
+      "graphite",
+      "shader"
+    ]
+  },
+  "construction": {
+    "category_weight": 10,
+    "entry_list": [
+      "treated_wood",
+      "multiblocks",
+      "balloon",
+      "crate",
+      "barrel",
+      "metalconstruction",
+      "metal_barrel",
+      "concrete",
+      "lighting",
+      "silo",
+      "tank"
+    ]
+  },
+  "energy": {
```

Comment:
Why does this category not have a weight? Same for heavy machines?

Reply:
Because those categories are created in code (and long before the JSON is loaded), so the weight in the JSON would just be ignored. I didn't want to put a value in here to prevent confusion along the lines of "I changed this value, but the order in the manual didn't change!".

Reply:
I see, fair enough. I suppose it would be nice if all categories loaded from this file for the sake of readability, but I recognize why order of operations does not allow for this.